tests/release/pipelines/fbc_release.go

```diff
@@ -100,17 +98,6 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 			_, err = devFw.AsKubeDeveloper.ReleaseController.CreateReleasePlan(fbcReleasePlanName, devNamespace, fbcApplicationName, managedNamespace, "true", nil, nil, nil)
 			Expect(err).NotTo(HaveOccurred())
 
-			sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
-			Expect(sourceAuthJson).ToNot(BeEmpty())
-
-			secret, err := devFw.AsKubeAdmin.CommonController.GetSecret(devNamespace, releasecommon.QuayTokenSecret)
-			if secret == nil || errors.IsNotFound(err) {
-				_, err = devFw.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(releasecommon.QuayTokenSecret, devNamespace, sourceAuthJson)
-				Expect(err).ToNot(HaveOccurred())
-				err = devFw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(devNamespace, releasecommon.QuayTokenSecret, constants.DefaultPipelineServiceAccount, true)
-				Expect(err).ToNot(HaveOccurred())
-			}
-
```

Comment:
Why is this being removed? Is it because that other service account will already have the required pull secrets?

Reply:
> Why is this being removed? Is it because that other service account will already have the required pull secrets?
Yes. The SA created automatically by build-service has the pull/push secrets.

Reply:
OK, thanks.